maloruellan/user

```diff
@@ -0,0 +1 @@
+maloruellan,Malo,Ruellan
```

Comment:
the file user seems good. But it should be in folder maloruellan/module1/user and not maloruellan/user